composer.json

```diff
@@ -13,7 +13,7 @@
         }
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "^8.0",
```

Comment:
Unfortunately, this constraint does not work because PHP does not follow [semantic versioning](https://semver.org).